engine/Shopware/Controllers/Backend/Base.php

```diff
@@ -146,6 +146,11 @@ public function getPaymentsAction()
         //select all shop as array
         $data = $query->getArrayResult();
 
+        // translate payments
+        /** @var $translationComponent Shopware_Components_Translation */
+        $translationComponent = Shopware()->Container()->get('translation');
```

Comment:
There's no need to access the DI container (`Shopware()->Container()`) directly, since any controller has a `get()` method for loading services.